src/desktop/apps/article/components/__tests__/App.test.js

```diff
@@ -70,4 +74,11 @@ describe('<App />', () => {
     rendered.find(ArticleLayout).length.should.equal(0)
     rendered.html().should.containEql('Video')
   })
+
+  it('renders a news article', () => {
+    const rendered = mount(<App article={NewsArticle} templates={{}} />)
+    rendered.find(Article).length.should.equal(1)
+    rendered.find(ArticleLayout).length.should.equal(0)
+    rendered.html().should.containEql('News')
+  })
```

Comment:
@eessex was there a test you had in mind in addition to this one?

Reply:
OOh somehow spaced on this -- yes looks good!